plugins/install-cert-manager.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: install-cert-manager
+spec:
+  platforms:
+  - head: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    sha256: c548f94d8efd1fb8aa35a54b7e5cdb64ae012447122d9704f3cee4163bbbe9c3
+    uri: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    files:
+    - from: "/*/install-cert-manager/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v1.0.1"
+  shortDescription: Install cert-manager helm chart
```

Comment:
I think just installing a helm chart doesn't warrant a plugin, especially because it's mostly an one-off operation and not repetitive. Not to mention, it won't scale to have a plugin per helm chart :)

Reply:
I use this one a lot because it also installs the cluster issuer which I always have to look up. Ya if it was just installing the chart there isn’t much value added.

Reply:
I think this is still a helm chart problem.

I've brought this up in https://github.com/jetstack/cert-manager/issues/259 so please participate in the discussion and support resolution of this issue there.

Reply:
It sounds like it is a helm problem, no clear simple resolution in sight other than having 2 helm charts for the issuers... I would still use my one-liner plugin to do it all.